Header header
string state   # "red", "not_red", or "unknown"
float32 confidence
geometry_msgs/Point center
